test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -236,7 +236,7 @@ public async Task InstallPackageToProjectsFromUI_PackageNamespace_WithSingleFeed
             <namespace id=""Test.*"" />
         </packageSource>
     </packageNamespaces>
-//</configuration>");
```

Comment:
were the `//`  causing test failures?

Reply:
Or actually, why weren't they causing test failures.

Reply:
It's just typo from my previous PRs.

Reply:
Does that cause an unparseable config?

Reply:
It looks xml is forgiving for closing tags. It would definitely fail for opening tag error.